### PR TITLE
Resume logging after log quota is hit

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots @cristiancavalli PTAL.

File: lib/v8debugapi.js [hunk 1]
```diff
@@ -205,12 +205,33 @@ module.exports.create = function(logger_, config_, fileStats_) {
...
+          logsThisSecond = 0;
+          timesliceEnd = currTime + 1000;
+        }
+        print(breakpoint.logMessageFormat,
+          breakpoint.evaluatedExpressions.map(JSON.stringify));
+        logsThisSecond++;
+        if (shouldStop()) {
+          delete listeners[num];
+        } else {
+          if (logsThisSecond > config.log.maxLogsPerSecond) {
```
Comment (cristiancavalli) [inline lib/v8debugapi.js]: Is the ceiling more than or `>=`?

Comment (matthewloring) [inline lib/v8debugapi.js]: Good catch. I have been thinking about this incorrectly. `>=` makes more sense.

Comment (matthewloring) [inline lib/v8debugapi.js]: Fixed.

**Comment by cristiancavalli** [comment]
LGTM pending CI

**Pull request merged**